modules/python/python-logmsg.c

```diff
@@ -254,12 +256,63 @@ _logmessage_get_keys_method(PyLogMessage *self)
   return keys;
 }
 
+static PyLogMessage *
+py_log_message_set_pri(PyLogMessage *self, PyObject *args, PyObject *kwrds)
+{
```

Comment:
Do we want to return with the message in case of this functions ? I guess it would support chaining function calls, but I do not know if it is aligned with python own style ?!

Reply:
Yes, chaining was my intention. I'm not familiar with Python conventions, so I can remove this if it is unconventional. :)